src/DynamoCoreWpf/UI/LibraryWrapPanel.cs

```diff
@@ -39,6 +39,13 @@ internal bool MakeOrClearSelection(NodeCategoryViewModel selectedClass)
                     classListView.SelectedIndex = itemIndex;
                     return true; // The call is handled.
                 }
+                else
```

Comment:
I noticed, that class button doesn't collapse, when it's clicked second time.
Fine, that noone has noticed it... :smirk: